Index: zmqpp-3.2.0/Makefile
===================================================================
--- zmqpp-3.2.0.orig/Makefile	2013-11-12 18:51:18.226008383 +0100
+++ zmqpp-3.2.0/Makefile	2013-11-12 18:53:47.882004460 +0100
@@ -124,11 +124,13 @@
 check: $(LIBRARY_SHARED) $(LIBRARY_ARCHIVE) test
 
 install:
+	mkdir -p $(INCLUDEDIR)/$(LIBRARY_DIR)
 	install -m 644 $(ALL_LIBRARY_INCLUDES) $(INCLUDEDIR)/$(LIBRARY_DIR)
+	mkdir -p $(LIBDIR)
 	install -m 755 $(BUILD_PATH)/$(LIBRARY_SHARED).$(VERSION_MAJOR) $(LIBDIR)/$(LIBRARY_SHARED).$(APP_VERSION)
 	ln -sf $(LIBRARY_SHARED).$(APP_VERSION) $(LIBDIR)/$(LIBRARY_SHARED).$(VERSION_MAJOR)
 	ln -sf $(LIBRARY_SHARED).$(APP_VERSION) $(LIBDIR)/$(LIBRARY_SHARED)
-	if [ -f $(BUILD_PATH)/$(CLIENT_TARGET) ]; then install -m 755 $(BUILD_PATH)/$(CLIENT_TARGET) $(BINDIR); fi
+	if [ -f $(BUILD_PATH)/$(CLIENT_TARGET) ]; then mkdir -p $(BINDIR); install -m 755 $(BUILD_PATH)/$(CLIENT_TARGET) $(BINDIR); fi
 	$(LDCONFIG)
 	@echo "use make installcheck to test the install"
 	
